### DISCUSSION TITLE
Anyone able to get this to actually work?

### DISCUSSION THREAD

**Discussion by HarleyCoops** [reorproject/reor#93] [General]
Never initializes, permanent spin. Never launches, no errors, anyone else?

**Comment by samlhuillier** [comment]
What hardware and OS do you have?

**Reply by HarleyCoops** [reply]
Windows 10 but on high end chips running a live bloomberg API and local LLMs. Not a process issue I don't think. Fully installs, logo on desktop but spins like this for hours, never comes back. 

![image](https://github.com/reorproject/reor/assets/76745467/307d963f-7aa2-4916-860b-28995ef56f2c)
